packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -305,8 +307,35 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
         _getListTile(
           appLocalizations.account_delete,
           () async {
+            await showDialog<SmoothAlertDialog>(
+                context: context,
+                builder: (BuildContext context) {
+                  return SmoothAlertDialog(
+                    title: appLocalizations.account_delete,
+                    body: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: <Widget>[
+                        Text(appLocalizations.account_delete_message),
+                        const SizedBox(
+                          height: 5,
+                        ),
+                        SmoothTextFormField(
+                            type: TextFieldTypes.PLAIN_TEXT,
+                            textInputType: TextInputType.text,
+                            controller: reasonController,
+                            hintText: appLocalizations.reason),
+                      ],
+                    ),
+                    positiveAction: SmoothActionButton(
+                      text: appLocalizations.okay,
+                      onPressed: () => Navigator.maybePop(context),
+                    ),
```

Comment:
There should be a `cancel` button.
In this specific case, the labels should be not confusing at all, like an obvious "Delete account" button.

Reply:
@monsieurtanuki @AshAman999 listing fixes I need to work on for this issue :
1. Cancel button for ux 
2. Should be `final String? reason = await showDialog<String>(`
3. Is there something I left here ?

Reply:
@Akashsri3bi Also:
* replacing that lazy "Okay" button with something more aggressive, like `account_delete`
* probably rendering `account_delete` and `cancel` buttons as vertical

Reply:
Okay @monsieurtanuki !